src/NUnitEngine/mock-assembly/mock-assembly.csproj

```diff
@@ -1,14 +1,11 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <RootNamespace>NUnit.Tests</RootNamespace>
-    <TargetFrameworks>net35;netcoreapp1.1;netcoreapp2.1;netcoreapp3.1</TargetFrameworks>
+    <TargetFrameworks>net35;netcoreapp2.1;netcoreapp3.1</TargetFrameworks>
     <SignAssembly>true</SignAssembly>
     <AssemblyOriginatorKeyFile>..\..\nunit.snk</AssemblyOriginatorKeyFile>
   </PropertyGroup>
-  <ItemGroup Condition="'$(TargetFramework)'=='netcoreapp1.1'">
-    <PackageReference Include="NUnit" Version="3.12.0" />
-  </ItemGroup>
-  <ItemGroup Condition="'$(TargetFramework)'!='netcoreapp1.1'">
+  <ItemGroup>
     <PackageReference Include="NUnit" Version="3.13.0-dev-06899" />
```

Comment:
This could be changed to the released version of NUnit 3.13 (but this should probably be in another PR)

Reply:
Yes, I agree.